frontend/packages/topology/src/components/graph-view/components/nodes/BaseNode.tsx

```diff
@@ -154,6 +154,7 @@ const BaseNode: React.FC<BaseNodeProps> = ({
             paddingX={8}
             paddingY={4}
             kind={kind}
+            data-test={`name-${element.getLabel()}`}
```

Comment:
I don't find any code which is looking for this `name-` data-test attribute?

Reply:
I was trying out something so this attribute was required but it didn't work out. I've not removed it because it can be used for future automation

Reply:
But then I would recommend another `data-test`. Please remove it from this PR when it is not used and then you can add it with a test scenario where it is used.

Reply:
Sure Christoph, will update it

Reply:
Updated